oonipipeline/tests/test_workflows.py

```diff
@@ -63,12 +63,12 @@ def test_make_file_entry_batch(datadir, db):
     )
 
     assert obs_msmt_count == 453
-    make_analysis_in_a_day(
+    make_analysis(
```

Comment:
After executing this function, can't we test something about its behavior?

Reply:
That was a good call. It lead to it spotting a bug in how the tests were run.

I am not adding too many behavioral checks atm because the analysis logic is still quite work in progress, but once it converges we should add more tests for the actual output values.